public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -153,18 +156,17 @@ export const AlertRuleForm: FC<Props> = ({ existing, prefill }) => {
 
   // only reset once we get some value in defaultDsAndQueries.queries, adding this value.
   useEffect(() => {
-    const shouldReset = !existing && !prefill && defaultDsAndQueries.queries;
+    const shouldReset = !existing && !prefill && defaultDsAndQueries.queries && !Boolean(defaultsInQueryParams);
     if (shouldReset) {
       reset({
         ...getDefaultFormValues(),
         queries: defaultDsAndQueries.queries,
         condition: 'C',
         ...defaultsInQueryParamsObject,
         type: RuleFormType.grafana,
-        evaluateEvery: evaluateEvery,
```

Comment:
Why was this line removed in this backport?

Reply:
there was a bug fix (not sure why is not in this branch) removing the evaluateEvery in here and in the array of dependencies

Reply:
I got a merge conflict on the array of dependencies

Reply:
https://github.com/grafana/grafana/pull/63763

Reply:
aha!, the backport was not merged yet https://github.com/grafana/grafana/pull/63794 @gillesdemey